flexinput/src/main/java/com/lytefast/flexinput/fragment/EmojiGridFragment.java

```diff
@@ -38,7 +35,16 @@
   private Unbinder unbinder;
 
   private Emoji.EmojiCategory emojiCategory;
+  private FlexInputFragment flexInputFrag;
 
+  @Override
+  public void onCreate(@Nullable final Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    Fragment parentFrag = getParentFragment();
```

Comment:
Each fragment does it differently (some don't keep the instance). I'm also not a fan of having base classes that do very little since inheritance is very restricting. Maybe when this FlexInput grows it'll make sense there.

e.g. if you have `MyAppBaseFragment` that you use everywhere you won't be able to use it here if we forced you to use our BaseFragment